modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/build.gradle

```diff
@@ -0,0 +1,10 @@
+//
+// Check GETTING_STARTED.markdown#liferay.workspace.target.platform.version
```

Comment:
So let's have a section in GETTING_STARTED for customzing the build with the root build.gradle. And in one section of the customizations possible we can have the BOM stuff.

Reply:
i.e. in this file i just want a link to the GETTING_STARTED, no commented out code (makes it hard to upgrade).